src/ms.tex

```diff
@@ -266,6 +266,46 @@ \subsection{Affiliated packages}
 Include a big table of all affiliated packages and references (as in v2.0
 paper).
 
+All affiliated packages are listed on the astropy 
+website\footnote{\url{https://www.astropy.org/affiliated}}. Since 
+\cite{astropy:2018}, there has been an explosion of gravitational-wave affiliated 
```

Comment:
I would not use the word "explosion", better "considerable increase" or "strong interest in implementing".